### PR TITLE
Backend: Save an image #66 added async function to save image to firebase

### PR DESCRIPTION
**Pull request by ajgcph2019**
# Description

Please provide a short summary explaining what this PR is about.
I have created an async function which takes a image as a parameter and saves in firebase storage and returns the download URL.
Fixes # (issue)

# How to test?
This story will be used by Tiago and Aswini for their story
Please provide a short summary how your changes can be tested?

# Checklist

- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/SaveImageToFirebase/SaveImageToFirebase.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    uploadTask.on(firebase.storage.TaskEvent.STATE_CHANGED,
+       () => {
+           const { downloadURL } = uploadTask.snapshot
+           return downloadURL
+       })
+      
+   }
+   return(
+    <button type="button" onClick ={handleSave}>Save </button>
+)
```
Comment (bytewiz) [inline src/client/components/SaveImageToFirebase/SaveImageToFirebase.js:19]: This formatted seems a bit off :) could you format the code?

File: src/client/components/SaveImageToFirebase/SaveImageToFirebase.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+const handleSave = () =>{
+    const bucketName = 'images';
+
+    const storageRef = firebase.storage().ref(`${bucketName}/${image}`)
+    const uploadTask = storageRef.put(image)
+    uploadTask.on(firebase.storage.TaskEvent.STATE_CHANGED,
+       () => {
+           const { downloadURL } = uploadTask.snapshot
+           return downloadURL
+       })
```
Comment (bytewiz) [inline src/client/components/SaveImageToFirebase/SaveImageToFirebase.js:14]: did you test this? :)

**Review by bytewiz — APPROVED** [review]

**Review by bytewiz — COMMENTED** [review]
Did you test that this works correctly?

**Review by bytewiz — CHANGES_REQUESTED** [review]
above

File: src/client/components/SaveImageToFirebase/SaveImageToFirebase.js (hunk 2 above)
Comment (ajgcph2019) [reply src/client/components/SaveImageToFirebase/SaveImageToFirebase.js]: Yes it is tested

File: src/client/components/SaveImageToFirebase/SaveImageToFirebase.js (hunk 1 above)
Comment (ajgcph2019) [reply src/client/components/SaveImageToFirebase/SaveImageToFirebase.js]: Yes I have pushed again after formatting

**Review by benna100 — APPROVED** [review]
There still are some unrelated changes but i will merge it

**Pull request merged**